styles/command-palette.less

```diff
@@ -0,0 +1,7 @@
+@import "ui-variables";
+
+// Highlight matched text
+.select-list .list-group .character-match {
```

Comment:
Good point - I should probably change this in fuzzy-finder too then.

(Although actually I was considering moving the styling to atom-space-pen-views... maybe a job for another day?)